examples/basics/data_rows.ipynb

```diff
@@ -2,18 +2,22 @@
   "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    local_data_path = '/tmp/test_data_row.txt'
</pre><p>Could you update to use an existent text file, eg . "https://lb-test-data.s3.us-west-1.amazonaws.com/text-samples/sample-text-1.txt"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>